### PR TITLE
[BrokenLinksH2] Update customer-key-manage.md

### PR DESCRIPTION
**Pull request by kturetz**
Corrected broken link on Line 103.

### CODE REVIEW

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kturetz, kindly commit the suggested change. Many thanks.

File: microsoft-365/compliance/customer-key-manage.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ To create a DEP to use with a mailbox, follow these steps:
    New-DataEncryptionPolicy -Name USA_mailboxes -Description "Root key for mailboxes in USA and its territories" -AzureKeyIDs https://contoso_EastUSvault02.vault.azure.net/keys/USA_key_01, https://contoso_CentralUSvault02.vault.azure.net/keys/USA_Key_02
    ```
 
-For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-data-encryptionpolicy).
+For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-dataencryptionpolicy?view=exchange-ps).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-dataencryptionpolicy).
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by kturetz** [comment]
**Global effort to fix broken links** 

@krowley 

 The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. Please review within five business days and merge, or comment in the PR with any changes you'd like to see. Thanks!

File: microsoft-365/compliance/customer-key-manage.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ To create a DEP to use with a mailbox, follow these steps:
    New-DataEncryptionPolicy -Name USA_mailboxes -Description "Root key for mailboxes in USA and its territories" -AzureKeyIDs https://contoso_EastUSvault02.vault.azure.net/keys/USA_key_01, https://contoso_CentralUSvault02.vault.azure.net/keys/USA_Key_02
    ```
 
-For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-data-encryptionpolicy).
+For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-dataencryptionpolicy).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md:103]:
```suggestion
For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-dataencryptionpolicy).
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**